src/Core/src/Hosting/AppHost.cs

```diff
@@ -0,0 +1,21 @@
+#nullable enable
+
+namespace Microsoft.Maui.Hosting
+{
+	public static class AppHost
+	{
+		public static IAppHost Current { get; internal set; } = null!;
+
+		public static IAppHostBuilder CreateDefaultBuilder()
```

Comment:
I moved this here because all the other versions of hosting, `WebHost` and `Host` were static classes that had this `CreateDefaultBuilder` method. We are now following that and adding a new `Current` because we need more flexibility due to our restricted environments. This includes stricter linking and as well as a non-pipeline/request model.

Reply:
it was like this on the start of this  :) so i agree